deduplicator/src/lib/utils.rs

```diff
@@ -161,20 +190,13 @@ where
     R: Fn(&Address) -> f64 + Clone + Send + 'static,
 {
     let input_conn = Connection::open(&path)?;
-    let nb_addresses = usize::try_from(input_conn.query_row(
-        "SELECT COUNT(*) FROM addresses;",
-        NO_PARAMS,
-        |row| row.get(0).map(|x: isize| x),
-    )?)
-    .expect("failed to count number of addresses");
 
     // Query list of addresses
     let mut stmt = input_conn.prepare("SELECT * FROM addresses;")?;
     let addresses = stmt
         .query_map(NO_PARAMS, |row| row.try_into())?
         .progress()
         .with_refresh_delay(refresh_delay)
-        .with_iter_size(nb_addresses)
```

Comment:
Out of curiosity, what is the behavior of the progress bar now that this iter size is no longer defined ?

Reply:
It will only display current iteration and speed approximation, it sounded fair to not bother keeping this behavior as importing SQLite files is not a feature we use (and we actually don't display any kind of progress for other formats).